starknet_py/tests/e2e/tests_on_networks/client_test.py

```diff
@@ -132,40 +132,41 @@ async def test_transaction_not_received_invalid_nonce(account_goerli_testnet):
 
 
 @pytest.mark.asyncio
-async def test_transaction_not_received_invalid_signature(account_goerli_testnet):
-    account = account_goerli_testnet
+async def test_transaction_not_received_invalid_signature(account_sepolia_testnet):
+    account = account_sepolia_testnet
     call = Call(
-        to_addr=int(EMPTY_CONTRACT_ADDRESS_GOERLI_TESTNET, 0),
+        to_addr=int(EMPTY_CONTRACT_ADDRESS_SEPOLIA_TESTNET, 0),
         selector=get_selector_from_name("empty"),
         calldata=[],
     )
     sign_invoke = await account.sign_invoke_v1(calls=call, max_fee=int(1e16))
     sign_invoke = dataclasses.replace(sign_invoke, signature=[0x21, 0x37])
-
-    with pytest.raises(ClientError, match=r"Signature.*is invalid") as exc:
+    # 0x0000000000004163636f756e743a20696e76616c6964207369676e6174757265 -> Account: invalid signature
```

Comment:
Sepolia returns only hex version of this string

Reply:
this is a regular string, you should be able to just convert its representation to get a proper human-readable message